qiskit/circuit/library/arithmetic/polynomial_pauli_rotations.py

```diff
@@ -174,19 +173,10 @@ def __init__(
             coeffs: The coefficients of the polynomial. ``coeffs[i]`` is the coefficient of the
                 i-th power of x. Defaults to linear: [0, 1].
             basis: The type of Pauli rotation ('X', 'Y', 'Z').
-            reverse: If True, apply the polynomial with the reversed list of qubits
-                (i.e. q_n as q_0, q_n-1 as q_1, etc).
             name: The name of the circuit.
         """
         # set default internal parameters
         self._coeffs = coeffs or [0, 1]
-        self._reverse = reverse
-        if self._reverse is True:
-            warnings.warn(
-                "The reverse flag has been deprecated. "
-                "Use circuit.reverse_bits() to reverse order of qubits.",
-                DeprecationWarning,
-            )
```

Comment:
For other reviewers: this was added in #4509 in June 2020, so well beyond the 3-month period.